<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addMalzemeDialog</class>
 <widget class="QDialog" name="addMalzemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kategoriler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Miktar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Miktar">
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_miktarType">
       <item>
        <property name="text">
         <string>Litre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Adet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gram</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KiloGram</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_kapat">
     <property name="text">
      <string>Kapat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_kapat</sender>
   <signal>clicked()</signal>
   <receiver>addMalzemeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
